src/components/DownloadMailingListButton.tsx

```diff
@@ -17,31 +16,40 @@ export function DownloadMailingListButton (props: DownloadButtonProps) {
         let email
         const propertyAddress = propertyObject.Deal_Name
         const ownerDetails = propertyObject.owner_details
-        const contact: OwnerType = ownerDetails[0]
-        let owner: OwnerType = ownerDetails[1]
+        const relatedContact: OwnerType = ownerDetails[0]
+        const owner: OwnerType = ownerDetails[1]
 
-        if (!contact && owner) {
+        let ownerOrRelatedContact: OwnerType = owner
+        if (!relatedContact && owner) {
             doNotMail = owner.Do_Not_Mail
             returnToSender = owner.Return_to_Sender
             postalAddress = owner.Postal_Address
             email = owner.Email
-        } else if (contact) {
-            doNotMail = contact.Do_Not_Mail
-            returnToSender = contact.Return_to_Sender
-            postalAddress = contact.Postal_Address
-            email = contact.Email
+            ownerOrRelatedContact = owner
+        } else if (relatedContact) {
+            doNotMail = relatedContact.Do_Not_Mail
+            returnToSender = relatedContact.Return_to_Sender
+            postalAddress = relatedContact.Postal_Address
+            email = relatedContact.Email
+            ownerOrRelatedContact = relatedContact
         }
 
+        const isDupe = ownerContactDupeRemoval.includes(`${ownerOrRelatedContact.Postal_Address}-${ownerOrRelatedContact.Name}`)
         if (!doNotMail || !returnToSender || !email) {
-            owner = owner ? (contact.Postal_Address ? contact : owner) : contact
-            const lastMailed = owner?.Last_Mailed || 'Last mailed has not been found'
-            csvRow = `"${owner?.Name}","${owner?.Contact_Type}","${postalAddress}","${owner?.Postal_Suburb}","${owner?.Postal_State}","${owner?.Postal_Postcode}","${propertyAddress}, ${lastMailed}\r\n`
-            csvRow = csvRow.replace(/null/g, '-')
+            const checker = null || undefined
```

Comment:
I'm not sure this does what you want it to do.

null || undefined === undefined

What is your desired logic?

Reply:
I'm also confused about this, I just want to make sure field wasn't empty but I did it in a weird way, fixed in next push

Reply:
Still an issue